src/GpuLidarSensor.cc

```diff
@@ -256,6 +256,14 @@ bool GpuLidarSensor::Update(const std::chrono::steady_clock::duration &_now)
     // Set the time stamp
     *this->dataPtr->pointMsg.mutable_header()->mutable_stamp() =
       msgs::Convert(_now);
+    // Set frame_id
+    for (auto i = 0; i < this->dataPtr->pointMsg.mutable_header()->data_size(); ++i) {
+      if (this->dataPtr->pointMsg.mutable_header()->data(i).key() == "frame_id"
```

Comment:
Style nits.

The linter is complaining here:

```
  /github/workspace/src/GpuLidarSensor.cc:260:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
  /github/workspace/src/GpuLidarSensor.cc:264:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
```

Also, please put the `{` in a new line

Reply:
lint is now happy but not so sure about the formating